core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/index.jelly

```diff
@@ -24,16 +24,17 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
-  <l:layout permission="${app.ADMINISTER}" title="${%Users}">
-    <st:include page="sidepanel.jelly" />
+  <l:layout permission="${app.ADMINISTER}" title="${%Users}" type="one-column">
```

Comment:
do you think the form could do with a max-width? it seems very wide now

<img width="1429" alt="image" src="https://user-images.githubusercontent.com/21194782/193417875-2c686fe2-2ef7-4b96-b642-296718e2cb80.png">

Reply:
Also does this button use primary button class? I don't see this form in the PR?

Reply:
> Also does this button use primary button class? I don't see this form in the PR?

The submit button used the `<f:submit />` component which currently uses the YUI classes, I've got a branch in the works to update that component 👍

Reply:
> do you think the form could do with a max-width? it seems very wide now
> 
> <img alt="image" width="1429" src="https://user-images.githubusercontent.com/21194782/193417875-2c686fe2-2ef7-4b96-b642-296718e2cb80.png">

Updated - 

![image](https://user-images.githubusercontent.com/43062514/193452751-4d6c718e-8580-4232-a63c-4eddbbfddb92.png)